<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cell_BkBrwClass</class>
 <widget class="QWidget" name="Cell_BkBrwClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cell_BkBrw</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QTableView" name="tableView">
     <property name="name" stdset="0">
      <string>tb</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="name" stdset="0">
      <string>bg</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="lb_qry">
        <property name="text">
         <string>搜索</string>
        </property>
        <property name="name" stdset="0">
         <string>lb</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox">
        <property name="name" stdset="0">
         <string>cbox</string>
        </property>
        <item>
         <property name="text">
          <string>书名</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>作者</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>类型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ID</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btn_brw">
        <property name="text">
         <string>申请借阅</string>
        </property>
        <property name="name" stdset="0">
         <string>btn</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_qry">
        <property name="name" stdset="0">
         <string>le</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_ref">
        <property name="text">
         <string>刷新页面</string>
        </property>
        <property name="name" stdset="0">
         <string>btn</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
